src/reference/glossary.rst

```diff
@@ -60,6 +60,12 @@ Glossary
 
       Some :term:`workflows<workflow>` produce a single, synonymous dataset, like Zika. Others, like seasonal flu, produce many datasets.
 
+   narrative
+
+      A Markdown file with additional syntax to specify a :term:`dataset` to display alongside the text.
```

Comment:
This is a perfect technical definition, but maybe not so relevant to new users?

Suggest adding something to describe the multi-page functionality of narratives (yes it's a single file, but is more of a slideshow to readers). Example:

> A method to convey information through multiple pages of content. Datasets can be displayed alongside text as interactive Auspice views. Saved as a Markdown file with extended syntax to support additional displays.

Reply:
Fair! Thanks for the nudge. Revised and repushed.